woocommerce/woocommerce-gutenberg-products-block
# Move merchant-side Checkout E2E tests to Playwright

## What

Moves merchant-side Checkout E2E tests to Playwright - we skipped them in puppeteer to get old E2Es working again.

Removes the old puppeteer merchant checkout tests.

I combined some tests to make it run a little faster and also ensured tests were not unnecessarily nested in describe blocks that don't add any value.

Related to https://github.com/woocommerce/woocommerce-blocks/issues/11833

## Why

To 1) re-enable a skipped test and 2) move away from puppeteer and onto Playwright

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Check playwright E2E tests pass, both `block_theme` and `classic_theme`


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Skipping
